.github/workflows/run_ert2_test_data_setups.yml

```diff
@@ -0,0 +1,50 @@
+name: Python package
+
+on: [push, pull_request]
+
+env:
+  ERT_SHOW_BACKTRACE: 1
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.7]
```

Comment:
Should perhaps add `3.6` as that is out target Python release internally at the moment?